template/AksApi.csproj

```diff
@@ -5,6 +5,8 @@
   </PropertyGroup>
   <!--#if (enableOpenApi) -->
```

Comment:
Think we might need to change this line for the inclusion of the ItemGroup tag: 
```
<!--#if (enableLogging or enableOpenApi) -->
<ItemGroup>
...
```
Then have separate conditions around the two sets of PackageReferences:
```
<!--#if (enableLogging) -->
<PackageReference ...LoggingPackages...>
...
```

Reply:
I've created two item groups, one for each. Tested with a build and it seems happy